core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -259,16 +259,29 @@ export class ContributionsAndReview
           this.activeTabType === this.TAB_TYPE_REVIEWS ? 'Review' : 'View'),
         translationWordCount: (
           this.isReviewTranslationsTab() && this.activeExplorationId) ? (
-            this.htmlLengthService.computeHtmlLength(
-              suggestion.change_cmd.content_html,
-              CALCULATION_TYPE_WORD)) : undefined
+            this.getTranslationContentLength(
+              suggestion.change_cmd.content_html)) : undefined
       };
 
       translationContributionsSummaryList.push(requiredData);
     });
     return translationContributionsSummaryList;
   }
 
+  getTranslationContentLength(contentHtml: string | string[]): number {
+    if (typeof contentHtml === 'string') {
+      return this.htmlLengthService.computeHtmlLength(
+        contentHtml, CALCULATION_TYPE_WORD);
+    } else if (Array.isArray(contentHtml)) {
```

Comment:
I think we need to throw and error in the `else` block?

Reply:
Done, Thanks.